app/views/confirmations/new.html.haml

```diff
@@ -1,7 +1,7 @@
 - content_for :header do
   Process a visit request
 
-- if prison_data(booked_visit)['canned_responses']
+- if booked_visit.prison_canned_responses
```

Comment:
For our next trick, perhaps we can rename `canned_responses` to something more descriptive: it doesn't seem awfully clear to me.